### PR TITLE
feat: update redirect to feed page for unauthenticated users

### LINKED ISSUE #1461: Feature: redirect user to /feed from homepage
### Type of feature

🍕 Feature

### Current behavior

When a user visits https://insights.opensauced.pizza, if they are not logged in, they are redirected to https://insights.opensauced.pizza/javascript/dashboard/filter/recent.

This change would redirect them to the highlights feed page at https://insights.opensauced.pizza/feed

### Suggested solution

Update existing redirects

### Additional context

_No response_

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [ ] I agree to follow this project's Contribution Docs

### PR DESCRIPTION
**Pull request by OgDev-01**
## Description
This PR updates the redirect from homepage to `/feed` when a user is not authenticated

## What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Related Tickets & Documents
Closes #1461 

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

### CODE REVIEW

**Comment by brandonroberts** [comment]
@OgDev-01 the e2e tests may need updating also

**Review by bdougie — APPROVED** [review]
Looks good. Take a look at that test.

File: components/organisms/UserSettingsPage/user-settings-page.tsx [hunk 1]
```diff
@@ -248,7 +248,7 @@ const UserSettingsPage = ({ user }: userSettingsPageProps) => {
               name="linkedin_url"
```
Comment (NsdHSO) [inline components/organisms/UserSettingsPage/user-settings-page.tsx:248]: Could we can revert these changes ???

File: components/organisms/UserSettingsPage/user-settings-page.tsx [hunk 2]
```diff
@@ -248,7 +248,7 @@ const UserSettingsPage = ({ user }: userSettingsPageProps) => {
...
             />
             <TextInput
-              className="bg-light-slate-4 text-light-slate-11 font-medium"
+              className="font-medium bg-light-slate-4 text-light-slate-11"
```
Comment (brandonroberts) [inline components/organisms/UserSettingsPage/user-settings-page.tsx:251]:
```suggestion
              className="bg-light-slate-4 text-light-slate-11 font-medium"
```

File: components/organisms/UserSettingsPage/user-settings-page.tsx [hunk 3]
```diff
@@ -374,7 +374,7 @@ const UserSettingsPage = ({ user }: userSettingsPageProps) => {
             </Button>
           </div>
           {!hasReports && (
-            <div className="flex flex-col gap-6 order-first md:order-last">
+            <div className="flex flex-col order-first gap-6 md:order-last">
```
Comment (brandonroberts) [inline components/organisms/UserSettingsPage/user-settings-page.tsx:377]:
```suggestion
            <div className="flex flex-col gap-6 order-first md:order-last">
```

**Review by brandonroberts — APPROVED** [review]
